src/components/formPost/formPost.tsx

```diff
@@ -0,0 +1,117 @@
+import React, { useState } from 'react';
+
+type User = {
+  id: number;
+  name: string;
+  username: string;
+  email: string;
+};
+
+type ToDo = {
+  id: number;
+  title: string;
+  completed: boolean;
+  userId: number | null;
+  user: User | null;
+};
+
+type Props = {
+  onSubmit: (newAdd: ToDo) => void;
+  users: User[];
+  heighestId: number;
```

Comment:
There is a typo in the prop name `heighestId`. It should be `highestId` to match the variable name used in other parts of the code.